src/components/playerstats/playerstats.js

```diff
@@ -191,6 +191,20 @@ function getDisplayBitrate(bitrate) {
     }
 }
 
+function getDisplayTranscodeFps(session, player) {
+    const mediaSource = playbackManager.currentMediaSource(player) || {};
+    const videoStream = (mediaSource.MediaStreams || []).find((s) => s.Type === 'Video') || {};
+
+    const originalFramerate = videoStream.AverageFrameRate;
+    const transcodeFramerate = session.TranscodingInfo.Framerate;
+
+    if (!originalFramerate) {
+        return `${session.TranscodingInfo.Framerate} fps`;
```

Comment:
You can reuse variable.
```suggestion
        return `${transcodeFramerate} fps`;
```